x/ibc/testing/chain.go

```diff
@@ -201,7 +201,9 @@ func (chain *TestChain) NextBlock() {
 		AppHash: chain.App.LastCommitID().Hash,
 		// NOTE: the time is increased by the coordinator to maintain time synchrony amongst
 		// chains.
-		Time: chain.CurrentHeader.Time,
+		Time:               chain.CurrentHeader.Time,
+		ValidatorsHash:     chain.Vals.Hash(),
+		NextValidatorsHash: chain.Vals.Hash(),
```

Comment:
I guess it's ok for now but it would be useful to test validator changes

Reply:
add to #6509 testing pkg wishlist